cluster-autoscaler/simulator/predicates.go

```diff
@@ -163,22 +160,11 @@ func NewPredicateChecker(kubeClient kube_client.Interface, stop <-chan struct{})
 	}, nil
 }
 
-// IsNodeReadyAndSchedulablePredicate checks if node is ready.
-func IsNodeReadyAndSchedulablePredicate(pod *apiv1.Pod, meta predicates.Metadata, nodeInfo *schedulernodeinfo.NodeInfo) (bool,
```

Comment:
TODO: verify if this doesn't break "GPU hack" where we override node conditions to pretend it's still booting up.